scipy/integrate/quadpack.py

```diff
@@ -560,6 +560,21 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
     romb : integrator for sampled data
     scipy.special : for coefficients and roots of orthogonal polynomials
 
+    Examples
+    --------
+
+    Calculate :math: double integration exp(-x*y)/y**5
+    Limits from 0 to inf
```

Comment:
This integral over `y` does not converge.
Maybe better write an exact formula in TeX notation.

Reply:
Can you please elaborate

Reply:
I actually suggest some simpler integral, which can be checked manually. E.g. (not necessarily this specific example):

Compute the moment of inertia of a square square plate with the axis of rotation at the center of the plate (the answer should be `$I = m (h^2 + w^2) / 12$`):

```
In [3]: I, err = dblquad(lambda x, y: x**2 + y**2, -1, 1, lambda x: -1, lambda x : 1)

In [4]: m, err = dblquad(lambda x, y: 1, -1, 1, lambda x: -1, lambda x: 1)

In [5]: I/m / (2**2 + 2**2)
Out[5]: 0.08333333333333333

```

Reply:
Done